test/test_gpu_tensorcore.cpp

```diff
@@ -4045,6 +4070,82 @@ TEST_F(NVFuserTest, FusionAmpereMMATNAlpha_CUDA) {
   TORCH_CHECK(cg_outputs[0].allclose(tref, 0.0001, 0.0001));
 }
 
+TEST_F(NVFuserTest, FusionAmpereMatmulEpilogue_CUDA) {
```

Comment:
Could you add a test case where there is both an epilogue unswizzle and a single->half casting fusion, and another kernel where there is a relu fusion?